torch/fx/experimental/rewriter.py

```diff
@@ -103,3 +105,20 @@ def __init__(self, orig):
     else:
         # Rewrite this single free function
         return AST_Rewriter().rewrite(cast(FunctionType, fn))
+
+def _copy_func(f, globals=None, module=None):
```

Comment:
Thanks Ansley. Currently I have no plans for the helper function, but I thought making it generic maybe helpful for others in the future.

I removed the checks related to `globals` and `module` in this commit: e52bb5b. I changed the function name from `_copy_func` to `_copy_func_and_overwrite_globals` because I feel that the function is now no longer generic in copying functions. However, if you prefer changing this back to `_copy_func` I can do that.

Reply:
OK. I renamed the helper function to `change_func_globals` and reduced its scope to be within the `rewrite` function only. Hope it makes more sense now.